config/helm/chart/default/templates/Common/extensions-opentelemetry-collector/clusterrole-extensions-opentelemetry-collector.yaml

```diff
@@ -44,6 +44,17 @@ rules:
       - get
       - list
       - watch
+  {{- if (eq (include "dynatrace-operator.openshiftOrOlm" .) "true") }}
+  - apiGroups:
+      - security.openshift.io
+    resourceNames:
+      - privileged
+      - nonroot-v2
```

Comment:
you don't need `nonroot-v2` as you don't set a custom `runAs` in the securityContext of the otel `StatefulSet`
(`restricted-v2` is enough in OCP 4.12 and later, and you don't need a role to use that)

Reply:
removed in dc0115b43